packages/react-storage/src/StorageManager/utils/uploadFile.ts

```diff
@@ -49,7 +50,7 @@ export function uploadFile({
       resumable: false,
       progressCallback,
       contentType,
-      provider,
+      provider: provider as `AWSS3`,
```

Comment:
Looks like we might be missing some commits maybe? Think @reesscot left a comment abt this in the code?

Reply:
I added the one on line 44. For some reason my editor didn't show an error on this one.

Reply:
yea  we need this for TS strict